Aladin-iOS/Aladin-iOS/Screens/BookDetail/BookDetailVC.swift

```diff
@@ -1068,4 +1074,41 @@ extension BookDetailVC {
             }
         }
     }
+    
+    func discount(origin: Int, discountRate: Int) -> Int{
+        let result : Int = origin * (100 - discountRate) / 100
+        return result
+    }
```

Comment:
할인 비율 가져오는 함수 따로 만드는거 너무 좋아요 잘하셨어여👍🏻
덧붙이자면 네트워크 익스텐션 보다는 위에 메소드 부분에 넣어줘도 될거같아요
함수 이름이 모호한 점이 있어서 이런 방식으로 고쳐도 좋을 것 같구 위에와 같이 컨벤션 지키는 습관..!!!!! 까지 하면 베스트임다
result에 타입 추론이 되기 때문에 Int 지정부분은 빼줘도 좋을 것 같아요 
```suggestion
    private func getDiscountPrice(origin: Int, discountRate: Int) -> Int {
        let result = origin * (100 - discountRate) / 100
        return result
    }
```

Reply:
컨벤션.. 주의하겠슴니당